whisper/benchmark.py

```diff
@@ -46,19 +46,26 @@ def everything():
 
 
 if __name__ == "__main__":
-
     # get command line arguments without 3rd party libraries
     # the command line argument to benchmark all models is "all"
     models = ["tiny"]
-    if len(sys.argv) > 1:
-        if sys.argv[1] == "--all":
-            models = ["tiny", "small", "medium", "large"]
+    for i, arg in enumerate(sys.argv):
+        if arg == "-all":
+            models = ["tiny", "small", "medium", "large-v1", "large-v3"]
+            break
+        elif arg in ("-m", "--models") and i + 1 < len(sys.argv):
+            models = sys.argv[i + 1].split(",")
```

Comment:
Can we use argparse here? It seems like it would simplify this a bit.

Reply:
Done 👍